demos/bert_named_entity_recognition_demo/python/bert_named_entity_recognition_demo.py

```diff
@@ -53,123 +57,102 @@ def build_argparser():
     args.add_argument("-d", "--device",
                       help="Optional. Target device to perform inference on."
                            "Default value is CPU", default="CPU", type=str)
+    args.add_argument('-nireq', '--num_infer_requests', help='Optional. Number of infer requests.',
+                      default=0, type=int)
+    args.add_argument('-nstreams', '--num_streams',
+                      help='Optional. Number of streams to use for inference on the CPU or/and GPU in throughput '
+                           'mode (for HETERO and MULTI device cases use format '
+                           '<device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>).',
+                      default='', type=str)
+    args.add_argument('-nthreads', '--num_threads', default=None, type=int,
+                      help='Optional. Number of threads to use for inference on CPU (including HETERO cases).')
     return parser
 
+
+def print_raw_results(score, filtered_labels_id, meta):
+    if not filtered_labels_id:
+        return
+    sentence, c_token_s_e = meta['sentence'], meta['c_token_s_e']
+    log.info('\t\tSentence: \n\t{}'.format(sentence))
+    visualized = set()
+    for id, label_id in filtered_labels_id:
+        word_s, word_e = c_token_s_e[id - 1]
+        if (word_s, word_e) in visualized:
+            continue
+        visualized.add((word_s, word_e))
+        word = sentence[word_s:word_e]
+        confidence = score[id][label_id]
+        tag = label_to_tag[label_id]
+        log.info('\n\tWord: {}\n\tConfidence: {}\n\tTag: {}'.format(word, confidence, tag))
+
+
 def main():
     args = build_argparser().parse_args()
 
-    # load vocabulary file for model
+    paragraphs = get_paragraphs(args.input)
+
+    preprocessing_start_time = perf_counter()
     vocab = load_vocab_file(args.vocab)
```

Comment:
We should treat the vocab file as label files for Image Models. So the best practice would be load it in the wrapper itself. Also wrapper should accept, not only path to file, but actual vocab as well

Reply:
I believe the wrapper is not the best place for loading a vocab (from a file or actual one). In _bert_question_answering_embedding_demo_, two wrappers `BertEmbedding` and `BertQuestionAnswering` require the vocab for setting special tokens. In this case the vocab will be loaded twice, and it looks odd.
Also, the vocab is used more often for tokenization of text/sentences, which is applied in the demo itself for now, while in the wrapper the vocab used only for special tokens mapping from vocab.